entities/symbol.py

```diff
@@ -1,6 +1,6 @@
 from .common import Name, Position, Rotation, Length, Description, Keywords, Author, Version, Created, Category, Deprecated, Polygon, Text
 from typing import List
-from common import indent
+from .helper import indent_entities
```

Comment:
This import sort order is not isort compliant :wink:

Reply:
Well you configured the CI to only check the top level files :wink:

Reply:
Fixed.